ci-operator/step-registry/telco5g/release-cluster/telco5g-release-cluster-ref.metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+	"path": "telco5g/release-cluster/telco5g-release-cluster-ref.yaml",
+	"owners": {
+		"approvers": [
+			"yuvalk",
+			"sshnaidm",
+			"lack",
+			"fedepaol",
+			"SchSeba",
+			"cynepco3hahue"
```

Comment:
artyom does not work for RH anymore, you can remove him

Reply:
ack